src/components/SubNav.tsx

```diff
@@ -0,0 +1,81 @@
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import {
+  compose,
+  height,
+  HeightProps,
+  margin,
+  MarginProps,
+  minHeight,
+  MinHeightProps,
+  minWidth,
+  MinWidthProps,
+  padding,
+  PaddingProps,
+  variant,
+  width,
+  WidthProps,
+} from '@modulz/radix-system';
+
+type SubNavProps = MarginProps & PaddingProps & HeightProps & MinHeightProps;
+
+const styleProps = compose(margin, padding, height, minHeight);
+
+export const SubNav = styled('nav')<SubNavProps>(
+  css({
+    height: 48,
+    display: 'flex',
+    boxShadow: '0 1px 0 0 #dcdcdc',
+    padding: 0,
+  }),
+  styleProps
+);
+
+type SubNavItemProps = PaddingProps & WidthProps & MinWidthProps & {
+  variant?: 'normal' | 'active';
+};
+
+const subNavItemStyleProps = compose(
+  padding,
+  minWidth,
+  width,
+);
+
+export const SubNavItem = styled('a')<SubNavItemProps>(
+  css({
+    alignItems: 'center',
+    backgroundColor: 'transparent',
+    border: 0,
+    boxSizing: 'border-box',
+    fontWeight: 600,
+    cursor: 'pointer',
+    display: 'flex',
+    paddingY: 0,
+    paddingX: 3,
+    textDecoration: 'none',
+    userSelect: 'none',
+    WebkitTapHighlightColor: 'rgba(0, 0, 0, 0)',
+    transition: 'transition: color 0.3s, background-color 0.3s',
+    '&::-moz-focus-inner': {
+      border: 0,
+    },
+    '&:hover': {
```

Comment:
I found that we miss to set hover styling when the item is active. I handled it in the Nav component https://github.com/containous/faency/pull/4/commits/9c661b8e3759e27c189f7543b7acda452d173902